ui/src/singletons/store.ts

```diff
@@ -246,6 +285,12 @@ const useStore = create<AppStore>()(
   )
 )
 
+export const useUndoRedo = () =>
```

Comment:
:memo: nitpick, but can we move this hook lower, next to the other exported hooks.